hw/ip/otbn/dv/rig/rig/configs/base.yml

```diff
@@ -12,6 +12,7 @@ gen-weights:
 
   # Generators that end the program
   ECall: 1
+  BadAtEnd: 5
```

Comment:
Are you intentionally giving this generator such a high weight, or is this a leftover from testing?

Reply:
Oops, thanks. Fixed.